src/persist-client/src/read.rs

```diff
@@ -633,7 +639,7 @@ where
         // NB: min_elapsed is intentionally smaller than the one in
         // maybe_heartbeat_reader (this is the preferential treatment mentioned
         // above).
-        let min_elapsed = PersistConfig::FAKE_READ_LEASE_DURATION / 4;
+        let min_elapsed = self.cfg.reader_lease_duration / 4;
         let elapsed_since_last_heartbeat = self.last_heartbeat.elapsed();
```

Comment:
semi-orthogonal thought: to fully bring reader leases into parity with writer leases, I'd love to see the `elapsed` calculation use `cfg.now()` so we can more easily play with timing. It's been nagging me for a bit that we handle reader and writer leases differently:

https://github.com/MaterializeInc/materialize/blob/73a16f3f83627220a89e8940efd15dcededd9ef5/src/persist-client/src/write.rs#L555-L566

Reply:
yeah, this is much better than how the read one was working. done!